service/templates/admin.html

```diff
@@ -911,6 +911,14 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 				                    <md-input-container class="control" style="width:45%; margin-left:50px;">
 										<label>Local storage size</label>
 										<input type="number" ng-model="Settings.amboy.local_storage_size">
+										<label>Group default workers</label>
```

Comment:
You've got some css problems 
![image](https://user-images.githubusercontent.com/29384396/61881651-fa705680-aec4-11e9-93a9-e43fb4c2e52d.png)

Reply:
fixed